src/GHC/Util/HsDecl.hs

```diff
@@ -1,12 +1,15 @@
 {-# LANGUAGE PackageImports #-}
 {-# LANGUAGE NamedFieldPuns #-}
 
-module GHC.Util.HsDecl (declName) where
+module GHC.Util.HsDecl (declName,isForD') where
 
 import "ghc-lib-parser" HsSyn
 import "ghc-lib-parser" OccName
 import "ghc-lib-parser" SrcLoc
 
+isForD' :: LHsDecl GhcPs -> Bool
+isForD' (LL _ ForD{}) = True; isForD' _ = False
```

Comment:
Why the ' after isForD?

Reply:
For the time being, I'm writing these ticks into all utilities that can end up cohabiting with HSE code in intermediate states as I convert modules containing before and after versions of a program in parallel - it's a device.  It's on the `isForD` for consistencies' sake at this time. When the conversion is done, the ticks will go away completely (so I hope you're not too fussed at this state of affairs today)!

Reply:
👍